Amharris/updating graph docs (#15821)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/delegatedadminaccessassignment.md

---
title: "delegatedAdminAccessAssignment resource type"
description: "Represents an assignment of access details to an access container."
author: "smrtsec"
ms.localizationpriority: medium
ms.prod: "gdap"
doc_type: resourcePageType
---

# delegatedAdminAccessAssignment resource type

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Represents an assignment of access details (containing roles) to an access container (like a security group). Once it's active, the members of the access container get access to the roles specified in the access details.

Reviewer: ```suggestion
Represents an assignment of access details (containing roles) to an access container (such as a security group). Once it is active, the members of the access container gain access to the roles specified in the access details.
```
